nighthawk/WORKSPACE

```diff
@@ -0,0 +1,22 @@
+workspace(name = "nighthawk")
+
+local_repository(
```

Comment:
We could also consume Envoy as a `git_repository` and not use submodules. To me, the advantage of submodules is the ability to go into the Envoy tree and hack it as needed, and be able to then submit these Envoy-side PRs upstream, so I think what you have here is the right choice.

Reply:
I feel the same way, being able to play around with and tweak the Envoy code base has turned out to be pretty useful.